test/en/en_time_later.test.js

```diff
@@ -69,12 +69,27 @@ test("Test - Single Expression", function() {
 });
 
 test("Test - Single Expression (Strict)", function() {
-
     var text = "15 minute after";
     var results = chrono.strict.parse(text, new Date(2012,7,10,12,14));
-    expect(results.length).toBe(0);
+    expect(results[0]).toBe(undefined);
```

Comment:
I found this to be more useful than the existing pattern because it'd then pretty print the first result to help in trying to figure out which parser was hit and such.

It'd probably be good to do this everywhere in the tests. I can make that change if you agree.